corehq/apps/domain/forms.py

```diff
@@ -548,16 +564,28 @@ def clean_default_geocoder_location(self):
             return data
         return json.loads(data or '{}')
 
-    def clean_logo(self):
-        logo = self.cleaned_data['logo']
-        if self.can_use_custom_logo and logo:
-            if logo.size > settings.MAX_UPLOAD_SIZE_ATTACHMENT:
+    def _clean_image(self, field_name, permission, error_message):
+        image = self.cleaned_data[field_name]
+        if permission and image:
```

Comment:
am I missing something or could this be `if permission and image and image.size > settings.MAX_UPLOAD_SIZE_ATTACHMENT:`